src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -101,6 +101,18 @@ export class MeetingManager implements AudioVideoObserver {
 
   simulcastEnabled: boolean = false;
 
+  skipAudioInputProviderInit: (() => void) | null = null;
```

Comment:
Do we need to do it this way? 
Can we just have one parameter such as setupDeviceConfig of type `DeviceLabels` which builders can pass via meetingConfig.  We can have a function in `meetingManager` such as `initDevices(deviceLabels: deviceLabels | deviceLabelTrigger)`
`AudioInputProvider`, `VideoInputProvider` can watch `setupDeviceConfig` flag. if it is None then do not do anything. If it changes to the corresponding value, then init.

Reply:
If we don't need to pass `DeviceLabel` via `MeetingConfig`, we can't use the subscribe pattern. Cause we have nothing to subscribe to. If we add something to subscribe, it could be quite similar with the these functions. Any better idea?